editor/tests/org.fusesource.ide.camel.editor.tests.integration/src/main/java/org/fusesource/ide/camel/editor/integration/properties/creators/NumberParameterPropertyUICreatorIT.java

```diff
@@ -39,6 +39,7 @@ public void setup() {
 		parameter = new Parameter();
 		parameter.setName("testParameterName");
 		parameter.setJavaType(Integer.class.getName());
+		parameter.setType("string");
```

Comment:
I think that in real world type will be always integer and not string

as it is the case for camel-servlet: 
uriParameter name="responseBufferSize" type="integer" javaType="java.lang.Integer"